python_modules/libraries/dagster-aws/dagster_aws/pipes/clients/emr_serverless.py

```diff
@@ -124,15 +125,15 @@ def run(
 
     def _enrich_start_params(
         self,
-        context: OpExecutionContext,
+        context: Union[OpExecutionContext, AssetExecutionContext],
         session: PipesSession,
         params: "StartJobRunRequestRequestTypeDef",
     ) -> "StartJobRunRequestRequestTypeDef":
         # inject Dagster tags
         tags = params.get("tags", {})
         tags = {
             **tags,
-            "dagster/run_id": context.run_id,
+            "dagster/run_id": context.run.run_id,
```

Comment:
AssetExecutionContext throws a deprecation warning if you call `context.run_id` but both Op and Asset context support `context.run.run_id`